roxie/ccd/ccdfile.cpp

```diff
@@ -288,7 +289,11 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
                         if (isCompressed && !isKey)
                             current.setown(createCompressedFileReader(f));
                         else
+                        {
                             current.setown(f->open(IFOread));
+                            if (current && blockedIOsize)
```

Comment:
future: we should probably also use blockIOsize for compressed files (if significantly above the compressed block size)